src/components/user/headerLogin/header.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './header.css';
+
+
+const Header = () => {
+
+    return (
+        <div className='header-main'>
+            <div className='header-main-second'>
+                <p className='title-logo'>🎁</p>
```

Comment:
@Dharmikpanchani 
Why are you not using the icon given in the design?

Reply:
sorry sir, 
I will tack care 
I updated Image Icon